app/models/manageiq/providers/amazon/instance_types.rb

```diff
@@ -1328,50 +1280,146 @@ module ManageIQ::Providers::Amazon::InstanceTypes
 
     "f1.2xlarge"  => {
       :name                    => "f1.2xlarge",
-      :family                  => "Unknown",
+      :family                  => "Accelerated computing",
       :description             => "F1 Double Extra Large",
       :memory                  => 122.0.gigabytes,
       :vcpu                    => 8,
       :ebs_only                => false,
-      :instance_store_size     => 480.0.gigabytes,
+      :instance_store_size     => 470.0.gigabytes,
       :instance_store_volumes  => 1, # SSD
       :architecture            => [:x86_64],
-      :virtualization_type     => [],
-      :network_performance     => :high,
```

Comment:
is `:network_performance` a free text? IIRC it should be rather a set of [:high, :low, etc]

Reply:
Similarly, there are new values breaking the original parsing logic which was
```
 -    o[:network_performance] = case i[:network_performance]
 -                              when '10 Gigabit', '20 Gigabit'
 -                                :very_high
 -                              else
 -                                i[:network_performance].downcase.tr(' ', '_').to_sym
 -                              end
```
Yes, AWS use free text for this field. So it's hard to parse it into structured type.

Reply:
I dont think we actually use this.